--- a/qjs.c
+++ b/qjs.c
@@ -104,6 +104,8 @@
 }
 
 /* also used to initialize the worker context */
+extern JSModuleDef *js_init_module_ppv_crypto(JSContext *ctx, const char *name);
+
 static JSContext *JS_NewCustomContext(JSRuntime *rt)
 {
     JSContext *ctx;
@@ -113,6 +115,7 @@
     /* system modules */
     js_init_module_std(ctx, "std");
     js_init_module_os(ctx, "os");
+    js_init_module_ppv_crypto(ctx, "ppv-crypto");
     return ctx;
 }
 
